prow/images/vulnerability-scanner/README.md

```diff
@@ -0,0 +1,22 @@
+# Docker Image containing node and go environments and Snyk application installed
+
+## Overview
+
+This folder contains the Golang Buildpack with nodejs and Snyk image that is based on the Golang Buildpack image. Use it to perform vulnerability scans.
+
+The image consists of:
+
+- Go 1.11.4
+- Dep 0.5.0
+- Gcloud 219.0.1
+- Docker 18.06.1*
+- nodejs
+- Snyk 1.134.2
+
+## Installation
+
+To build the Docker image, run this command:
```

Comment:
Is this sufficient? If other teams are to use this tool then you probably should provide more details. If just runs on Prow then ignore this comment.

Reply:
It runs only on Prow.